dashboard/modules/log/log_manager.py

```diff
@@ -88,7 +88,10 @@ async def stream_logs(
             keep_alive=keep_alive,
             lines=options.lines,
             interval=options.interval,
-            timeout=options.timeout,
+            # If we keepalive logs connection, we shouldn't have timeout
+            # otherwise the stream will be terminated forcefully
+            # after the deadline is expired.
+            timeout=options.timeout if not keep_alive else None,
```

Comment:
i think there are multiple timeout: connection time/receive timeout. Does the client allow us to specify connection timeout? For streaming we probably want to handle connection timeout?

Reply:
Right, I think we could specify (connect timeout, read timeout) as a pair. https://requests.readthedocs.io/en/latest/user/advanced/#timeouts

Reply:
This is using gRPC, not requests. I think connection timeout should be automatically handled by gRPC